build.gradle

```diff
@@ -1,11 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
Yep. Was planning on bumping this up from mavenCentral. Thanks for getting to this.